packages/react-native-web/src/vendor/hash/index.js

```diff
@@ -11,6 +11,8 @@
  * @param {string} str ASCII only
  * @param {number} seed Positive integer only
  * @return {number} 32-bit positive integer hash
+ *
+ * @flow
```

Comment:
Is it necessary to flow type this file?

Reply:
It helped flow another export in another file I believe (always returns string)